manage/config-dialog.js

```diff
@@ -197,8 +197,12 @@ function configDialog(style) {
     renderValues();
   }
 
+  function isNullOrUndefined(value) {
+    return value === null || value === undefined;
```

Comment:
That would require adding an eslint exception. Anyway we decided to never rely on the implicit quirks on loose comparison so it'd really weird to use it. 

I also think this isNullOrUndefined looks like a kludge which could be implemented better e.g. `!isNaN(Number(va.min))` - this can be extracted to a function like isNumber, of course. For `step` and default value `isFinite` should be added to the check, otherwise `1e11111` will produce `Infinity` which is only valid for max (and `-Infinity` is valid for `min`).

Reply:
I ended up with:

```js
function isNumber(value) {
  return value !== '' && !isNaN(Number(value));
}
```

since `Number('')` returns `0`.

Reply:
`Number(null)` also returns 0. Use something like `typeof value === "number" && !isNaN(value)`

Reply:
But `typeof '1' === 'number'` is `false`. So maybe just include a `null` check.